### PR TITLE
chore: integrate quests

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-9573

### CODE REVIEW

**Review by tcheee — APPROVED** [review]

**Review by admsk2 — APPROVED** [review]

File: src/app/ui/quests/QuestMissionPage.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+'use client';
+
+import { QuestsMissionPage } from 'src/components/Quests/QuestPage/QuestsMissionPage';
+import { JUMPER_LOYALTY_PATH } from 'src/const/urls';
+
+const QuestPage = ({ quest, url }: any) => {
+  return (
+    <QuestsMissionPage quest={quest} baseUrl={url} path={JUMPER_LOYALTY_PATH} />
```
Comment (oktapodia) [inline src/app/ui/quests/QuestMissionPage.tsx:8]: This const migh tnot be necessary anymore

File: src/components/Quests/AvailableMissionsList/AvailableMissionsList.tsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                quest.attributes?.CustomInformation?.['rewardType'];
+              const rewardRange =
+                quest.attributes?.CustomInformation?.['rewardRange'];
+              const chains = quest.attributes.CustomInformation?.['chains'];
+              const claimingIds =
+                quest.attributes?.CustomInformation?.['claimingIds'];
+              const rewardsIds =
+                quest.attributes?.CustomInformation?.['rewardsIds'];
+
+              //todo: exclude in a dedicated helper function
```
Comment (oktapodia) [inline src/components/Quests/AvailableMissionsList/AvailableMissionsList.tsx:122]: agreed

File: src/components/Quests/Rewards/RewardsCarousel.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+interface RewardsCarouselProps {
+  hideComponent: boolean;
+  rewardAmount: number;
+  accumulatedAmountForContractBN: string;
+  isMerklSuccess: boolean;
+  proof: string[];
+}
+
+const CLAIMING_CONTRACT_ADDRESS = '0x3Ef3D8bA38EBe18DB133cEc108f4D14CE00Dd9Ae';
```
Comment (oktapodia) [inline src/components/Quests/Rewards/RewardsCarousel.tsx:30]: Might need to be dynamic isn't it?

**Review by oktapodia — CHANGES_REQUESTED** [review]
Couple of comments

File: src/components/Quests/Rewards/RewardsCarousel.tsx (hunk 3 above)
Comment (tcheee) [reply src/components/Quests/Rewards/RewardsCarousel.tsx:30]: Yes done in the next PR: https://github.com/jumperexchange/jumper-exchange/pull/1268

**Review by oktapodia — APPROVED** [review]
LGTM

**Pull request merged**